docker/docker_transport_test.go

```diff
@@ -163,6 +221,16 @@ func TestReferenceNewImage(t *testing.T) {
 	})
 	require.NoError(t, err)
 	defer img.Close()
+
+	ref, err = ParseReference("//quay.io/libpod/busybox" + UnknownDigestSuffix)
+	require.NoError(t, err)
+	_, err = ref.NewImage(context.Background(), &types.SystemContext{
```

Comment:
(Absolutely non-blocking: Use a single variable to define this `SystemContext` instead of copy&pasting? Also elsewhere. This works just fine.)

Reply:
Fixed